packages/components/src/__future__/Workflow/_docs/Workflow.mdx

```diff
@@ -0,0 +1,172 @@
+import { ArgTypes, Canvas, Meta } from "@storybook/blocks"
+import { Installation, KaioNotification, ResourceLinks,LinkTo } from "../../../../../../storybook/components"
+import { InlineNotification } from "@kaizen/notification"
+import * as Workflow from "./Workflow.stories"
+import * as WorkflowFooter from "./WorkflowFooter.stories"
+import * as WorkflowHeader from "./WorkflowHeader.stories"
+
+<Meta title="Pages/Workflow" />
+
+# Workflow
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/Workflow"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<InlineNotification type="cautionary" persistent>
```

Comment:
```suggestion
<InlineNotification type="informative" persistent>
```

I don't want this to blend with the KAIO notification, I think informative would be more appropriate anyways.

Reply:
True, the contrast is a little nicer on the eyes as well
![Screenshot 2023-06-20 at 5 00 58 pm](https://github.com/cultureamp/kaizen-design-system/assets/36558508/1579e173-02b7-498f-9f8e-0d13f4a255e2)